src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -64,7 +64,7 @@ public static function register($mode = false)
                     $group = 'remaining';
                 }
 
-                if (isset($mode[0]) && '/' === $mode[0] && preg_match($mode, $class.'::'.$method)) {
+                if (isset($mode[0]) && '/' === $mode[0] && preg_match($mode, $msg)) {
```

Comment:
I don't understand why this preg_match is there. Is it an undocumented feature of the helper ?